plugin/iswap.vim

```diff
@@ -8,6 +8,10 @@ command ISwap lua require('iswap').iswap()
 command ISwapWith lua require('iswap').iswap_with()
 command ISwapWithRight lua require('iswap').iswap_with('right')
 command ISwapWithLeft lua require('iswap').iswap_with('left')
+command IMove lua require('iswap').imove()
+command IMoveWith lua require('iswap').imove_with()
+command IMoveWithRight lua require('iswap').imove_with('right')
```

Comment:
I Feel like `IMoveWith{Right,Left}` are a bit silly since it's just ISwapWith

Reply:
but might be nice to have

Reply:
I also wasn't sure about that. IMoveRight and IMoveLeft are actually logically the same as ISwapLeft and ISwapRight. I added those commands just for symmetry and also since iswap and imove share the same function underneath it just works. I think it's fine, seems unnecessary to explicitly prohibit it in the Lua api but the commands could be removed